examples/protocol_buffers/BUILD.bazel

```diff
@@ -23,7 +23,10 @@ ts_proto_library(
 proto_library(
     name = "car_proto",
     srcs = ["car.proto"],
-    deps = [":tire_proto"],
+    deps = [
+        ":tire_proto",
+        "@com_google_googleapis//google/api:annotations_proto",
```

Comment:
You added the dependency to the wrong proto_library
See https://github.com/bazelbuild/rules_nodejs/pull/2276/commits/fc78dd68b863138b7c2a0169c5267e93f6885456 for a fix

Reply:
Doh. Thanks.  Fixed in PR, the test failures are now more in line with what others have been reporting.